django/db/backends/base/features.py

```diff
@@ -137,6 +137,9 @@ class BaseDatabaseFeatures:
     # which can't do it for MyISAM tables
     can_introspect_foreign_keys = True
 
+    # Features related to database level on delete
+    has_on_delete_db_default = True
```

Comment:
This comment means nothing, TBH.
```suggestion
    # Does the backend support ...
    supports_on_delete_db_default = True
```